src/weighted_matrix.cpp

```diff
@@ -13,44 +18,59 @@ using namespace Rcpp;
 // [[Rcpp::export]]
 arma::mat weight_matrix_cpp(List features, int m, int n, NumericVector w){
 
-    // Define variables
+    // Note that Z is a n x (m + 1) matrix, where m is the number
+    // of unique features. All elements in the first column are equal to 1.
+    // For j > 0, Z(i, j) = 1 if and only if feature j is present in
+    // the ith combination of features. In example, if Z(i, j) = 1 we know that
+    // j is present in l[i].
```

Comment:
Should be "j is present in features[i]" ???

Reply:
Yes!

Reply:
Fixed with a5bea03.